src/utils_request.js

```diff
@@ -45,26 +68,11 @@ export const REQUEST_AS_BROWSER_DEFAULT_OPTIONS = {
  * All `options` not recognized by this function are passed to it,
  * so see it for more details.
  *
- * @param options.url
- *  URL of the target endpoint. Supports both HTTP and HTTPS schemes.
- * @param [options.method=GET]
- *  HTTP method.
- * @param [options.headers]
- *  Additional HTTP headers to add. It's only recommended to use this option,
- *  with headers that are typically added by websites, such as cookies. Overriding
- *  default browser headers will remove the masking this function provides.
- * @param [options.languageCode=en]
- *  Two-letter ISO 639 language code.
- * @param [options.countryCode=US]
- *  Two-letter ISO 3166 country code.
- * @param [options.isMobile]
- *  If `true`, the function uses User-Agent of a mobile browser.
- * @param [options.abortFunction]
- *  Function accepts `response` object as a single parameter and should return true or false.
- *  If function returns true request gets aborted. This function is passed to the
- *  (@apify/http-request)[https://www.npmjs.com/package/@apify/http-request] NPM package.
+ * @param {RequestAsBrowserOptions} options All `requestAsBrowser` configuration options.
  *
- * @return {Promise<http.IncomingMessage|stream.Readable>}
+ * @return {Promise<http.IncomingMessage|stream.Readable>} This will typically be a
```

Comment:
oddly enough, `await requestAsBrowser()` is returning `any`. I'm trying to pinpoint why

Reply:
I spent more than a day working this out. Please let us know!

Reply:
I think you might be able to force it to return just `IncomingMessage`. That gives the correct return type and the interfaces are identical AFAIK.

Reply:
I couldn't make it work at all as well, the types need to be imported, like the rest, without the `///` at the top

Reply:
there's another issue, depending on the options "stream" (and it's set as "useStream") on default options. when `stream = true`, the return is a stream with `IncomingMessage` attached to it, from [here](https://github.com/apifytech/http-request/blob/master/src/add_response_properties_to_stream.js), so it's an intersection (and I think it's beyond the capabilities of JSDoc). 
otherwise, the `IncomingMessage` has a `body` property attached, that is "string".

Reply:
Any progress on this one?